.github/workflows/dvc-diff.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
     # workaround from https://github.com/iterative/cml/issues/1377
     - name: Setup NodeJS
-      uses: actions/setup-node@v3.8.1
+      uses: actions/setup-node@v4.0.0
       with:
         node-version: '16'
```

Comment:
Looking at the newer comments in https://github.com/iterative/cml/issues/1377 around Jul 2023, it seems like this workaround applied in https://github.com/GenericMappingTools/pygmt/pull/2549 might not be needed anymore if we set some `write` permissions for this dvc-diff GitHub Action?